content/docs/get-started/aws/_index.md

```diff
@@ -13,6 +13,14 @@ aliases: ["/docs/quickstart/aws/"]
 
 {{< cloud-intro "AWS" >}}
 
-Let's start with installing Pulumi.
+- Configure a Pulumi Community Edition account
+- Deploy a S3 Bucket
+- Modify the bucket to include a KMS key
```

Comment:
I didn't know what a KMS key was when I started working here, and I suspect many/most won't, either. If we're going to keep leading with KMS key, we should explain it more generally, like "create a storage bucket on Amazon S3" and  "protect the bucket's contents with server-side encryption".

Reply:
👍